src/kontrol/kdist/hevm.md

```diff
@@ -0,0 +1,40 @@
+### Hevm Success Predicate
```

Comment:
I know it's a nitpick, but it looks like `hevm` is spelled in all lowercase in their documentation and repo. Can I please ask you to use the same here and in the next line? Or, if you'd prefer, all uppercase `HEVM` might be fine too.
```suggestion
### hevm Success Predicate
```